### PR TITLE
RC2 updates for Secret manager

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@blowdart  please approve security changes
@danroth27 | @rachelappel  please provide squirrel

File: aspnet/security/app-secrets.rst [hunk 1]
```diff
@@ -14,74 +14,89 @@ This tutorial shows how your application can securely store and access secrets i
...
 
 Secret Manager
 ^^^^^^^^^^^^^^
 
 The Secret Manager tool provides a more general mechanism to store sensitive data for development work outside of your project tree. The Secret Manager tool is a project tool that can be used to store secrets for a `.NET Core`_ project during development. With the Secret Manager tool you can associate app secrets with a specific project and share them across multiple projects.
 
-.. note:: The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only.
+.. warning:: The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only. The keys and values are stored in a JSON configuration file in the user profile directory.
 
 Installing the Secret Manager tool
```
Comment (blowdart) [inline aspnet/security/app-secrets.rst]: Sub heading maybe, to nest it under Secret Manager?

Comment (blowdart) [inline aspnet/security/app-secrets.rst]: Sub heading maybe, to nest it under Secret Manager?

**Comment by blowdart** [comment]
I'm fine with this.

File: aspnet/security/app-secrets.rst (hunk 1 above)
Comment (Rick-Anderson) [inline aspnet/security/app-secrets.rst]: @blowdart  Given that Non-encryption is the #1 customer complaint, I want a warning  that visually stands out
image

>I'm fine with this.
I still need a [:]shipit[:}

Comment (blowdart) [inline aspnet/security/app-secrets.rst]: Oh no, I mean the actual heading, Installing the Secret Manager tool. Right now it's at the same level as the  Secret Manager heading, to mean it should be a level under that.

**Comment by Rick-Anderson** [comment]
@blowdart  my bad. Yes, that's an improvement. Squirrel now?

**Comment by blowdart** [comment]
:shipit:

**Pull request merged**